Sport1Player/SwiftFile.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  SwiftFile.swift
+//  Sport1Player
+//
+//  Created by Oliver Stowell on 05/09/2019.
+//  Copyright © 2019 Applicaster Ltd. All rights reserved.
+//
+
+import Foundation
```

Comment:
A comment of why this file is needed should be good, so no other developers in the future will delete the file.